spark/v3.5/spark-runtime/NOTICE

```diff
@@ -82,42 +245,7 @@ NOTICE file:
 
 --------------------------------------------------------------------------------
 
-This binary artifact includes Carrot Search Labs HPPC with the following in its
```

Comment:
If this is no longer present, can you verify there are no classes in the Jar and remove relocations for it?

Reply:
```
$ jar tvf iceberg-spark-runtime-3.5_2.12-1.8.0-SNAPSHOT.jar|grep -i carrot
$
```
I don't see any carrot class in the jar.